### ISSUE TITLE
Changing color of datasets in the map plotter has no effect

### ISSUE THREAD

**Issue by Acefall** [data-hydenv/hydenv-database#11]
After selecting a new color for the data set in the map plotter the color on the map stays the same.
![mapColorChange](https://user-images.githubusercontent.com/76061447/104166519-3a9a0080-53fb-11eb-8d54-918c5349984f.png)

**Comment by mmaelicke** [comment]
Yeah, can confirm that changing the color for map plots does not work. But for line and bar it does work. Can you confirm @Acefall ?

I think this is a low priority issue, but I will keep it open. Thanks for reporting.

**Comment by Acefall** [comment]
Yes works in the bar plot.
